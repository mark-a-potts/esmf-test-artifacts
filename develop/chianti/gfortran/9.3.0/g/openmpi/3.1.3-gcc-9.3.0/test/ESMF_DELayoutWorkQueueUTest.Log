build time -- 2021-04-23 12:43:04.760193
20210423 124750.154 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124750.154 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124750.154 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124750.154 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124750.154 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124750.154 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124750.154 INFO             PET0 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124750.154 INFO             PET0 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124750.154 INFO             PET0 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124750.154 INFO             PET0 ESMF_COMM                   : openmpi
20210423 124750.154 INFO             PET0 ESMF_MOAB                   : enabled
20210423 124750.154 INFO             PET0 ESMF_LAPACK                 : enabled
20210423 124750.154 INFO             PET0 ESMF_NETCDF                 : enabled
20210423 124750.154 INFO             PET0 ESMF_PNETCDF                : disabled
20210423 124750.154 INFO             PET0 ESMF_PIO                    : enabled
20210423 124750.154 INFO             PET0 ESMF_YAMLCPP                : enabled
20210423 124750.154 INFO             PET0  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20210423 124750.154 INFO             PET0  NUMBER_OF_PROCESSORS           4
20210423 124750.155 INFO             PET0  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20210423 124750.155 INFO             PET0  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20210423 124750.155 INFO             PET0  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20210423 124750.155 INFO             PET0  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20210423 124756.168 INFO             PET0  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20210423 124756.168 INFO             PET0  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20210423 124756.169 DEBUG            PET0 vmkt_create()#200 Pthread: service=0 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 20971520 bytes
20210423 124756.169 DEBUG            PET0 vmkt_create()#200 Pthread: service=0 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 20971520 bytes
20210423 124756.169 DEBUG            PET0 vmkt_create()#200 Pthread: service=0 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 20971520 bytes
20210423 124756.169 DEBUG            PET0 vmkt_create()#200 Pthread: service=0 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 20971520 bytes
20210423 124756.169 INFO             PET0  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20210423 124758.711 INFO             PET0  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20210423 124758.712 INFO             PET0  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20210423 124758.712 INFO             PET0  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20210423 124758.712 INFO             PET0 Finalizing ESMF
20210423 124750.154 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124750.154 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124750.154 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124750.154 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124750.154 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124750.154 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124750.154 INFO             PET1 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124750.154 INFO             PET1 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124750.154 INFO             PET1 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124750.154 INFO             PET1 ESMF_COMM                   : openmpi
20210423 124750.154 INFO             PET1 ESMF_MOAB                   : enabled
20210423 124750.154 INFO             PET1 ESMF_LAPACK                 : enabled
20210423 124750.154 INFO             PET1 ESMF_NETCDF                 : enabled
20210423 124750.154 INFO             PET1 ESMF_PNETCDF                : disabled
20210423 124750.154 INFO             PET1 ESMF_PIO                    : enabled
20210423 124750.154 INFO             PET1 ESMF_YAMLCPP                : enabled
20210423 124750.154 INFO             PET1  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20210423 124750.154 INFO             PET1  NUMBER_OF_PROCESSORS           4
20210423 124750.155 INFO             PET1  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20210423 124750.155 INFO             PET1  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20210423 124754.168 INFO             PET1  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20210423 124754.168 INFO             PET1  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20210423 124756.168 INFO             PET1  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20210423 124756.168 INFO             PET1  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20210423 124756.169 DEBUG            PET1 vmkt_create()#200 Pthread: service=1 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 4194304 bytes
20210423 124756.169 DEBUG            PET1 vmkt_create()#200 Pthread: service=1 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 4194304 bytes
20210423 124756.169 INFO             PET1  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20210423 124758.645 INFO             PET1  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20210423 124758.645 INFO             PET1  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20210423 124758.646 INFO             PET1  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20210423 124758.646 INFO             PET1 Finalizing ESMF
20210423 124750.154 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124750.154 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124750.154 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124750.154 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124750.154 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124750.154 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124750.154 INFO             PET2 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124750.154 INFO             PET2 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124750.154 INFO             PET2 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124750.154 INFO             PET2 ESMF_COMM                   : openmpi
20210423 124750.154 INFO             PET2 ESMF_MOAB                   : enabled
20210423 124750.154 INFO             PET2 ESMF_LAPACK                 : enabled
20210423 124750.154 INFO             PET2 ESMF_NETCDF                 : enabled
20210423 124750.154 INFO             PET2 ESMF_PNETCDF                : disabled
20210423 124750.154 INFO             PET2 ESMF_PIO                    : enabled
20210423 124750.154 INFO             PET2 ESMF_YAMLCPP                : enabled
20210423 124750.154 INFO             PET2  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20210423 124750.154 INFO             PET2  NUMBER_OF_PROCESSORS           4
20210423 124750.155 INFO             PET2  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20210423 124750.155 INFO             PET2  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20210423 124756.168 INFO             PET2  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20210423 124756.168 INFO             PET2  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20210423 124756.168 INFO             PET2  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20210423 124756.168 INFO             PET2  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20210423 124756.169 DEBUG            PET2 vmkt_create()#200 Pthread: service=1 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 4194304 bytes
20210423 124756.169 DEBUG            PET2 vmkt_create()#200 Pthread: service=1 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 4194304 bytes
20210423 124756.169 INFO             PET2  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20210423 124758.672 INFO             PET2  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20210423 124758.672 INFO             PET2  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20210423 124758.672 INFO             PET2  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20210423 124758.672 INFO             PET2 Finalizing ESMF
20210423 124750.154 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124750.154 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124750.154 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124750.154 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124750.154 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124750.154 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124750.154 INFO             PET3 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124750.154 INFO             PET3 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124750.154 INFO             PET3 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124750.154 INFO             PET3 ESMF_COMM                   : openmpi
20210423 124750.154 INFO             PET3 ESMF_MOAB                   : enabled
20210423 124750.154 INFO             PET3 ESMF_LAPACK                 : enabled
20210423 124750.154 INFO             PET3 ESMF_NETCDF                 : enabled
20210423 124750.154 INFO             PET3 ESMF_PNETCDF                : disabled
20210423 124750.154 INFO             PET3 ESMF_PIO                    : enabled
20210423 124750.154 INFO             PET3 ESMF_YAMLCPP                : enabled
20210423 124750.154 INFO             PET3  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20210423 124750.154 INFO             PET3  NUMBER_OF_PROCESSORS           4
20210423 124750.155 INFO             PET3  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20210423 124750.155 INFO             PET3  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20210423 124750.155 INFO             PET3  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20210423 124750.155 INFO             PET3  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20210423 124756.168 INFO             PET3  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20210423 124756.168 INFO             PET3  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20210423 124756.169 DEBUG            PET3 vmkt_create()#200 Pthread: service=1 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 4194304 bytes
20210423 124756.169 DEBUG            PET3 vmkt_create()#200 Pthread: service=1 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 4194304 bytes
20210423 124756.169 INFO             PET3  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20210423 124758.676 INFO             PET3  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20210423 124758.676 INFO             PET3  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20210423 124758.676 INFO             PET3  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20210423 124758.676 INFO             PET3 Finalizing ESMF
