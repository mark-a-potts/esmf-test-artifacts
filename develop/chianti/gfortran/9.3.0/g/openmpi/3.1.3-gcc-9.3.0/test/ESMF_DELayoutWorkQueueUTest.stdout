build time -- 2021-04-23 12:43:04.760193
  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  NUMBER_OF_PROCESSORS           4
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
 <round 1> PET            0  time:    8.9098000898957253E-005
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
 <round 1> PET            3  time:    9.5523893833160400E-005
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
 <round 1> PET            1  time:    4.0133672561496496     
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
 <round 1> PET            2  time:    6.0133892379235476     
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
 *** hi from mygcomp_setvm_withthreads ***
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
 *** hi from mygcomp_setvm_withthreads ***
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
 *** hi from mygcomp_setvm_withthreads ***
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
 *** hi from mygcomp_setvm_withthreads ***
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
 <round 2> PET            1  time:    2.4761316201183945     
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
 <round 2> PET            2  time:    2.5025721502024680     
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
 <round 2> PET            3  time:    2.5068131971638650     
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
 <round 2> PET            0  time:    2.5423394420649856     
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PET           1  Test Elapsed Time    6016.9600000000000       msec.
  PET           3  Test Elapsed Time    6015.7070000000003       msec.
  PET           2  Test Elapsed Time    6016.0539999999992       msec.
  PET           0  Test Elapsed Time    16039.967999999999       msec.
